lib/state/wallet/logic.dart

```diff
@@ -1793,4 +1794,54 @@ class WalletLogic extends WidgetsBindingObserver {
       launchUrl(Uri.parse(uri), mode: LaunchMode.externalApplication);
     } catch (_) {}
   }
+
+  void setWalletActionsLoading(bool loading) {
+    _state.setWalletActionsLoading(loading);
+  }
+
+  Future<void> evaluateWalletActions() async {
+    _state.setWalletActionsLoading(true);
+
+    _state.walletActions = [];
+
+    List<ActionButton> actionsToAdd = [];
+
+    final showVouchers = selectShowVouchers(_state);
+    final showMinter = selectShowMinter(_state);
+    final showPlugins = selectShowPlugins(_state);
+
+    if (showVouchers) {
+      actionsToAdd.add(ActionButton(
+        label: 'Vouchers',
+        buttonType: ActionButtonType.vouchers,
+      ));
+    }
+
+    if (showMinter) {
+      actionsToAdd.add(ActionButton(
+        label: 'Minter',
+        buttonType: ActionButtonType.minter,
+      ));
+    }
+
+    if (showPlugins) {
+      actionsToAdd.add(ActionButton(
+        label: 'Plugins',
+        buttonType: ActionButtonType.plugins,
+      ));
+    }
+
+    if (actionsToAdd.length > 1) {
+      actionsToAdd.add(ActionButton(
+        label: 'More',
+        buttonType: ActionButtonType.more,
+      ));
+    }
+
+    _state.walletActions = actionsToAdd;
+
+    await delay(const Duration(milliseconds: 500));
```

Comment:
I'm guessing that this had something to do with the animation or the fact that the values hadn't updated yet. 

If you make the changes above, you shouldn't need this.